### DISCUSSION TITLE
Feedback for User Input lesson

### DISCUSSION THREAD

**Discussion by jcollard** [CaptainCoderOrg/IntroToCSharpSite#111] [Page Feedback]
Feedback discussion for the **User Input** lesson: https://csharp-develop.captaincoder.org/lessons/variables/user-input

**Comment by cloudthepain** [comment]
**Pre-knowledge Check Section:**

-   "...know and where you have room to grow and learn." 
- It should be learn and grow.

---

- "The goal here is to see where you have room to grow and learn." 
- How about something like "The goal here is to explore the concepts at your pace and allow your curiosity to guide your learning."
- You repeat the previous line which doesn't feel right.
- I think the question order should be going from easier to harder.
- What is a variable, What is a string, what does it mean to declare a variable, What is a method?

**Reply by jcollard** [reply]
Great feedback! Thanks.

I changed the text to:

```
        Start by trying to answer the questions below. Don't be tempted to peek
        ahead to find the answers. It is okay if you don't know the answer yet.
        The goal is to see what you already know and where you have room to
        learn and grow.
```

What do you think of that?

**Reply by cloudthepain** [reply]
Yeah, that works a lot better and reads a lot cleaner.
